scripts/perf/scheduler_ops_metrics.py

```diff
@@ -71,7 +74,7 @@ def print_stats(self):
             .filter(TI.dag_id.in_(DAG_IDS))
             .all()
         )
-        successful_tis = filter(lambda x: x.state == State.SUCCESS, tis)
+        successful_tis = [x for x in tis if x.state == State.SUCCESS]
```

Comment:
Thanks for the comment @bolkedebruin! Without this change, the script fails with the following error at line 95 on Python3, since `filter` returns an iterator object rather than a list:

```
Traceback (most recent call last):
  File "scripts/perf/scheduler_ops_metrics.py", line 206, in <module>
    main()
  File "scripts/perf/scheduler_ops_metrics.py", line 202, in main
    job.run()
  File "/home/sekikn/dev/incubator-airflow/airflow/jobs.py", line 202, in run
    self._execute()
  File "/home/sekikn/dev/incubator-airflow/airflow/jobs.py", line 1584, in _execute
    self._execute_helper(processor_manager)
  File "/home/sekikn/dev/incubator-airflow/airflow/jobs.py", line 1714, in _execute_helper
    self.heartbeat()
  File "scripts/perf/scheduler_ops_metrics.py", line 134, in heartbeat
    self.print_stats()
  File "scripts/perf/scheduler_ops_metrics.py", line 95, in print_stats
    if len(tis) > len(successful_tis):
TypeError: object of type 'filter' has no len()
```

Reply:
`list(filter(...))` perhaps then?